dotCMS/src/main/resources/dotmarketing-config.properties

```diff
@@ -824,3 +824,11 @@ dartsass.compiler.deprecation.warnings.from.dependencies=false
 #analytics.app.write.url=http://jitsu
 #analytics.app.read.url=http://cube
 analytics.accesstoken.renewjob.cron=0 0/1 * * * ?
+
+## By default, We're now removing Content-Types  asynchronously, if you want to go back to the old behavior, set this property to false
+#CONTENT_TYPE_DELETE_ASYNC=true
```

Comment:
There is a typo on this property as in code it is called `DELETE_CONTENT_TYPE_ASYNC`

Reply:
This is what the config file looks like now
```
## By default, We're now removing Content-Types  asynchronously, if you want to go back to the old behavior, set this property to false
#DELETE_CONTENT_TYPE_ASYNC=true

## By default, Asynchronous Content-Type remove is carried out by a Quartz Job
## If You want to use the thread pool directly outside the Quartz Job, set this property to false
## This property is only effective if DELETE_CONTENT_TYPE_ASYNC=true
#DELETE_CONTENT_TYPE_ASYNC_WITH_JOB=true

```
By default the new mechanism is enabled to work using a Job however that text basically shows that it can disconnected by un commenting the property